### PR TITLE
rm: fix rm'ing a file

### PR DESCRIPTION
**Pull request by giuseppe**
regression introduced with 7dcef0769b1521b8f14bdee58339c26140466c85

Reported here: https://bodhi.fedoraproject.org/updates/FEDORA-2019-fff1ded16e#comment-958825

As part of the PR, tag also another release

### CODE REVIEW

**Comment by giuseppe** [comment]
please hold, I am doing some changes..

**Comment by giuseppe** [comment]
changes done

**Comment by giuseppe** [comment]
@rhatdan PTAL

File: tests/fedora-installs.sh [hunk 1]
```diff
@@ -1,10 +1,12 @@
...
+#!/bin/bash
+
+set -xeuo pipefail
 
 mkdir lower upper workdir merged
 
 fuse-overlayfs -o lowerdir=lower,upperdir=upper,workdir=workdir,suid,dev merged
 
-docker run --rm -ti -v merged:/merged fedora dnf --installroot /merged --releasever 29 install -y glibc-common gedit
+docker run --rm -ti -v merged:/merged fedora dnf --installroot /merged --releasever 30 install -y glibc-common gedit
```
Comment (rhatdan) [inline tests/fedora-installs.sh]: Not for this PR, but can we change these to use Podman?

**Comment by rhatdan** [comment]
LGTM

**Pull request merged**